src/documents/ChangeApplierAPI.md

```diff
@@ -446,29 +453,21 @@ These are not recommended for typical users of the framework, and are not stable
 
 ### Instantiating a ChangeApplier ###
 
-Instantiating a ChangeApplier manually is possible but is not recommended for general use. In particular, manual instantiation of a ChangeApplier will not allow access to the powerful [Model Relay](ModelRelay.md) framework features, which require the ChangeApplier to be properly situated in an IoC component tree. However, it's possible to imagine some cases where authors of "superframeworks" might find this facility useful. In Infusion 1.5 there are several variants for construction of ChangeAppliers since the implementation is in transition from the old non-relay-aware applier to the new model relay implementation.
-
-```javascript
-var applier = fluid.makeChangeApplier(model, options); // currently produces an "old-style" applier
-
-    OR
-var applier = fluid.makeHolderChangeApplier(holder, options); // currently produces an "old-style" applier
-    OR
-var applier = fluid.makeNewChangeApplier(holder, options); // produces a new "relay-aware" applier
-```
-
-In every case the `options` argument is optional. Where the parameter `model` appears it directly represents the model to be managed by the applier. Where parameter `holder` appears, it represents instead an object which is expected _to be a container for the model_ where the model itself is held in this object at the member named `model`. In framework use this argument will actually hold the owning component itself, although the applier makes no use of any properties of this object other than the one held at `model`.
+Instantiating a ChangeApplier manually is no longer recommended in current versions of the framework. Its implementation is tightly bound into its location in 
+an IoC component tree and should be constructed by the IoC system itself.
 
 Under the old ChangeApplier semantics, the `model` handle was closed over and remained constant for the lifetime of the applier. The framework contained utilities such a `fluid.model.copyModel` which assisted the user in working with a model with changing contents whilst keeping its base reference constant. Under the new ChangeApplier semantics, the `model` base reference is not stable and in fact starts by holding the value `undefined` as every model component initialises.
```

Comment:
Do we still need to retain this reference to the "old ChangeApplier semantics," since this is in fact now the "old old ChangeApplier semantic." In the spirit of 2.0 as a shedding of legacy concepts, can we also ditch this documentation or is it still crucial?